test/data/manifests/fedora-coreos-container.mpp.yaml

```
@@ -89,6 +89,7 @@ pipelines:
             sysroot:
               readonly: false
               bootloader: none
+            bls_append_except_default: true
```

to illustrate.. once you make the suggested changes abovie it would look like this I think:

```suggestion
            sysroot:
              readonly: false
              bootloader: none
              bls_append_except_default: grub_users=""
```

setting `grub_users=""` with the quotes in there might be tricky so play around with the yaml to figure out how to get it passed all the way through.

---

Just setting `grub_users=""` as is seems to work in the yaml since it automatically puts an escape character before each double quote.